### PR TITLE
LPS-89168 JavaDocs

### PR DESCRIPTION
**Pull request by gabrielwas**

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataEngineRequestExecutor.java [hunk 1]
```diff
@@ -21,33 +21,56 @@
...
 	public DEDataDefinition map(DDMStructure ddmStructure)
-		throws DEDataDefinitionFieldsDeserializerException {
+		throws PortalException {
+
+		List<DEDataDefinitionField> deDataDefinitionFields = deserialize(
+			ddmStructure.getDefinition());
 
 		DEDataDefinition deDataDefinition = new DEDataDefinition();
 
+		deDataDefinition.setDEDataDefinitionFields(deDataDefinitionFields);
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataEngineRequestExecutor.java]: @gabrielwas i believe we need to sort these set's.

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionGetRequestExecutor.java [hunk 2]
```diff
@@ -14,33 +14,22 @@
...
 /**
  * @author Jeyvison Nascimento
  */
 public class DEDataDefinitionGetRequestExecutor {
 
 	public DEDataDefinitionGetRequestExecutor(
-		DDMStructureLocalService ddmStructureLocalService,
-		DEDataDefinitionFieldsDeserializerTracker
-			deDataDefinitionFieldsDeserializerTracker) {
+		DEDataEngineRequestExecutor deDataEngineRequestExecutor,
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionGetRequestExecutor.java]: @gabrielwas sort this too

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionGetRequestExecutor.java [hunk 3]
```diff
@@ -14,33 +14,22 @@
...
+		DDMStructureLocalService ddmStructureLocalService) {
 
+		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionGetRequestExecutor.java]: and this one

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionListRequestExecutor.java [hunk 4]
```diff
@@ -36,15 +30,12 @@
 public class DEDataDefinitionListRequestExecutor {
 
 	public DEDataDefinitionListRequestExecutor(
-		DDMStructureService ddmStructureService,
-		DEDataDefinitionFieldsDeserializerTracker
-			deDataDefinitionFieldsDeserializerTracker,
-		Portal portal) {
+		DEDataEngineRequestExecutor deDataEngineRequestExecutor,
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionListRequestExecutor.java]: sort

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionGetRequestExecutor.java [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+		_ddlRecordSetLocalService = ddlRecordSetLocalService;
+		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
+	}
+
+	public DEDataRecordCollectionGetResponse execute(
+			DEDataRecordCollectionGetRequest deDataRecordCollectionGetRequest)
+		throws Exception {
+
+		long deDataRecordCollectionId =
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionGetRequestExecutor.java]: We dont need to declare this variable to be used only un the method above. We can call this method inside get record set.

File: modules/apps/data-engine/data-engine-test/src/testIntegration/java/com/liferay/data/engine/service/test/DEDataDefinitionServiceTest.java [hunk 6]
```diff
@@ -880,8 +881,8 @@ public void testGetBySiteMember() throws Exception {
 			DEDataDefinitionSaveResponse deDataDefinitionSaveResponse =
 				_deDataDefinitionService.execute(deDataDefinitionSaveRequest);
 
-			expectedDEDataDefinition.setDEDataDefinitionId(
-				deDataDefinitionSaveResponse.getDEDataDefinitionId());
+			expectedDEDataDefinition =
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-test/src/testIntegration/java/com/liferay/data/engine/service/test/DEDataDefinitionServiceTest.java]: No need to create this variable if we only use it in one Assert.

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Pull request closed without merging**